internal/runners/activate/activate.go

```diff
@@ -336,3 +347,12 @@ func warningForAdministrator(out output.Outputer) {
 		))
 	}
 }
+
+func activatedProjectName() (string, error) {
```

Comment:
Nit: I think it would be more intuitive if this was called something like `parentNamespace()` and returned the namespace type rather than a string.